cmd/prometheus/main.go

```diff
@@ -191,6 +192,9 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 			case "auto-gomaxprocs":
 				c.enableAutoGOMAXPROCS = true
 				level.Info(logger).Log("msg", "Automatically set GOMAXPROCS to match Linux container CPU quota")
+			case "scrape-rules":
+				c.enableScrapeRules = true
+				level.Info(logger).Log("msg", "Experimental scrape rules enabled")
```

Comment:
It appears that this is only for recording rules, is it worth mentioning it here?

Reply:
Thanks, I updated the description to reflect that. We can also change the feature flag name to `scrape-recording-rules` if that makes it more clear what the feature is about.